float64 xPos
float64 yPos
float64 zOrient
float65 wOrient
---